tests/conf.js

```diff
@@ -131,7 +133,7 @@ const login = browser => {
   browser.driver.findElement(by.id('login')).click();
   // Login takes some time, so wait until it's done.
   const bootstrappedCheck = () =>
-    element(by.css('body.bootstrapped')).isPresent();
+    element(by.id('messages-tab')).isPresent();
```

Comment:
Angular no longer adds classes to `body`, instead it adds them to the main wrapper `.app-root`. I think we can just replace references to `body` with `.app-root` (in this case `.app-root.bootstrapped`).

Reply:
done 👍